ddclient

```diff
@@ -493,6 +493,10 @@ my %variables = (
     'dondominio-common-defaults'       => {
         'server'	      => setv(T_FQDNP,  1, 0, 1, 'dondns.dondominio.com', undef),
     },
+    'mydns-common-defaults'   => {
+        'server'          => setv(T_FQDNP,  1, 0, 1, 'www.mydns.jp',      undef),
+        'max-interval'    => setv(T_DELAY,  0, 0, 1, interval('6d'),      0),
+    },
```

Comment:
Can we move those variables into the service specific ones?

Reply:
Yes. No problem.